### PR TITLE
veda ffmpeg role

### PR DESCRIPTION
**Pull request by muhammad-ammar**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [x] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

File: playbooks/roles/edx_ansible/templates/update.j2 [hunk 1]
```diff
@@ -61,6 +61,7 @@ repos_to_cmd["insights"]="$edx_ansible_cmd insights.yml -e 'INSIGHTS_VERSION=$2'
 repos_to_cmd["ecommerce"]="$edx_ansible_cmd ecommerce.yml -e 'ECOMMERCE_VERSION=$2'"
 repos_to_cmd["discovery"]="$edx_ansible_cmd discovery.yml -e 'DISCOVERY_VERSION=$2'"
 repos_to_cmd["notifier"]="$edx_ansible_cmd notifier.yml -e 'NOTIFIER_VERSION=$2'"
+repos_to_cmd["veda_ffmpeg"]="$edx_ansible_cmd veda_ffmpeg.yml -e 'VEDA_FFMPEG_VERSION=$2'"
```
Comment (muhammad-ammar) [inline playbooks/roles/edx_ansible/templates/update.j2]: Changes in this file will be removed. These are just for testing.

File: playbooks/edx-east/veda_ffmpeg.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+- name: Execute veda_ffmpeg role
+  hosts: all
+  become: True
+  gather_facts: True
+  roles:
+    - veda_ffmpeg
```
Comment (muhammad-ammar) [inline playbooks/edx-east/veda_ffmpeg.yml]: This file will be removed. These are just for testing.

**Comment by muhammad-ammar** [comment]
@feanil This is ready for review.

File: playbooks/roles/veda_ffmpeg/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+# wiki:       https://openedx.atlassian.net/wiki/display/OpenOPS
+# code style: https://openedx.atlassian.net/wiki/display/OpenOPS/Ansible+Code+Conventions
+# license:    https://github.com/edx/configuration/blob/master/LICENSE.TXT
+#
+##
+# Defaults for veda_ffmpeg role
+#
+
+VEDA_FFMPEG_GIT_URL: 'https://github.com/yro/v_videocompile'
+veda_ffmpeg_root: '/opt/veda_ffmpeg'
```
Comment (feanil) [inline playbooks/roles/veda_ffmpeg/defaults/main.yml]: Why is this not in a standard `/edx/app/` directory?

**Review by feanil — COMMENTED** [review]
This role needs to be added to the base `edx_continuous_integration.yml` playbook as well.  If you want it installed by default on sandboxes.

Do you have a sandbox with this installed?

File: playbooks/roles/veda_ffmpeg/defaults/main.yml (hunk 3 above)
Comment (muhammad-ammar) [reply playbooks/roles/veda_ffmpeg/defaults/main.yml]: @feanil We used this path to clone the repo to install the pkg. There are other existing roles using path for similar reasons.

**Comment by muhammad-ammar** [comment]
> This role needs to be added to the base edx_continuous_integration.yml playbook as well. If you want it installed by default on sandboxes.

I believe this role should be used as dependency for a relevant video pipeline role. It should not be installed by default.

**Comment by muhammad-ammar** [comment]
@feanil Yes, we have a [sandbox](ma-video-ffmpeg.sandbox.edx.org). [Sandbox build](https://tools-edx-jenkins.edx.org/job/Sandboxes/job/CreateSandbox/6990/consoleFull)

**Comment by muhammad-ammar** [comment]
@feanil But [these changes](https://github.com/yro/v_videocompile/pull/3) are required in v_videocompile repo. Waiting for greg to review and merge those changes. I have built the sandbox with [these changes](https://github.com/yro/v_videocompile/pull/3), so you can do testing on sandbox.

File: playbooks/roles/veda_ffmpeg/defaults/main.yml (hunk 3 above)
Comment (feanil) [reply playbooks/roles/veda_ffmpeg/defaults/main.yml]: Can you add a comment above this line then to explain that.

**Review by feanil — APPROVED** [review]

**Pull request merged**